common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -154,6 +160,25 @@ public void setTenantId(String tenantId) {
         mTenantId = tenantId;
     }
 
+    public static String getAudienceFromTenantId( final String tenantId) {
```

Comment:
I'm wondering if we should have a helper method takes in whole authority as opposed to just tenant id and then the helper method returns the audience.

So `getAudienceFromAuthority(authority)`

I think this should simplify the invocation in the Broker PR.

Reply:
[fbe8a84](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2495/commits/fbe8a841d3d18134489e28e519829cdbeb0fdc98)